### PR TITLE
Dtd 826 stub

### PR DESCRIPTION
**Pull request by vasanthavijaya**

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -25,3 +25,5 @@ GET           /test-only/requests
 DELETE        /test-only/request/:requestId                                                                 uk.gov.hmrc.debttransformationstub.controllers.TimeToPayTestController.deleteTTPRequest(requestId: String)
 GET           /test-only/request/:requestId                                                                 uk.gov.hmrc.debttransformationstub.controllers.TimeToPayTestController.getTTPRequest(requestId: String )
 POST          /test-only/response                                                                           uk.gov.hmrc.debttransformationstub.controllers.TimeToPayTestController.createTTPRequests
+
+POST          /individuals/debts/field-collections/templates                                                uk.gov.hmrc.debttransformationstub.controllers.DebtManagementAPITemplatesTestController.fieldCollectionsTemplates()
```
Comment (rogerball) [inline conf/app.routes:29]: should this be in /test-only/... format?

File: app/uk/gov/hmrc/debttransformationstub/controllers/DebtManagementAPITemplatesTestController.scala [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+  def fieldCollectionsTemplates(): Action[templateRequest] =
+    Action.async(parse.tolerantJson[templateRequest]) { request =>
+      if (appConfig.isPollingEnv)
+        pollingService.insertRequestAndServeResponse(Json.toJson(request.body), request.uri).map {
+          case Some(response) => Status(response.status.getOrElse(200))(response.content)
+          case None => ServiceUnavailable
+        }
+      else
+        environment.getExistingFile(s"$basePath/dm.template/template.json") match {
```
Comment (rogerball) [inline app/uk/gov/hmrc/debttransformationstub/controllers/DebtManagementAPITemplatesTestController.scala:49]: Consider renaming to indicate it is for fc...

File: conf/app.routes (hunk 1 above)
Comment (felixgb) [reply conf/app.routes:29]: Right now the routes in the `test-only` format are the routes for the bridge tool, the "normal" stub routes follow this format I think. Maybe move it above the bridge tool routes?

File: app/uk/gov/hmrc/debttransformationstub/controllers/DebtManagementAPITemplatesTestController.scala (hunk 2 above)
Comment (vasanthavijaya) [reply app/uk/gov/hmrc/debttransformationstub/controllers/DebtManagementAPITemplatesTestController.scala]: renamed file as suggested

File: conf/app.routes (hunk 1 above)
Comment (vasanthavijaya) [reply conf/app.routes]: Moved url to DM level to differentiate from bridge tool url's

**Pull request merged**